### ISSUE TITLE
Technical spec for minting runes

### ISSUE THREAD

**Issue by 0xbrito** [casey/runestone#19]
Can we discuss about it? I think that this is a priority.

**Comment by casey** [comment]
I personally usually write code before writing a spec. I find that I'm much more likely to fully flesh out the protocol and come up with the best implementation through coding. So I'll be implementing first, and then coming up with a spec afterwards. Of course, a motivated individual could try to write the spec in parallel. But there's a risk that the spec will need to be amended.

**Comment by Psifour** [comment]
@0xbrito This is a duplicate of #13 

Please remember to use the search bar to make sure that an existing discussion is not already underway before creating a new issue. This makes sure that the maximum number of people will see your messages as many of us only have notifications on for issues we expect to lead to discussions.

**Comment by 0xbrito** [comment]
@Psifour Sorry about that, closing this
